openproblems/tasks/regulatory_effect_prediction/methods/beta.py

```diff
@@ -226,6 +226,6 @@ def _atac_genes_score(adata, top_genes=500, threshold=1):
     code_url="http://cistrome.org/BETA",
     image="openproblems-python-extras",
 )
-def beta(adata, n_top_genes=500, threshold=1):
-    adata = _atac_genes_score(adata, top_genes=n_top_genes, threshold=threshold)
```

Comment:
also here... do we want explicit parameter passing in the function header or just implicit passing? Here it also looks like you're using defaults suddenly instead of the specified parameters.

Reply:
With these parameters we are just setting the same defaults in two different places -- happy to put it back in if we think it would be useful down the road to be able to change these values.

Reply:
As above, i think the `my_param` setup would be cool